source/assets/components/recipeCard/recipeCard.js

```diff
@@ -172,26 +172,33 @@ class recipeCard extends HTMLElement {
       if (dietary['dairy-free']) { this.shadowRoot.getElementById('dairy-free').removeAttribute('hidden') }
     }
 
-    // //determine the text on button
-    // const favoriteButton = this.shadowRoot.querySelector('button');
-    // const favImg = this.shadowRoot.querySelector('#fav-btn');
-    // if(myStorage.getItem(title) != undefined){
-    //   favoriteButton.textContent = "Remove the Favorite"
-    //   favImg.setAttribute('src', "./assets/icons/favorite/favorite-yellow.png")
-    // }
+    //Determine whether or not the recipe is favorite before
+    //If it is then it should change the icon to red heart
+    const favoriteButton = this.shadowRoot.querySelector('button');
+    const favImg = this.shadowRoot.querySelector('button img');
+    if(myStorage.getItem(title) != undefined){
+      favImg.setAttribute('src', "./assets/icons/favorite/favorite-red.png")
+      favImg.setAttribute('alt', 'unfavorite')
+    }
     
-    // //set favorite Button functionality
-    // favoriteButton.addEventListener('click', () => {
-    //   if(favoriteButton.textContent == "Favorite the Recipe"){
-    //     myStorage.setItem(title,JSON.stringify(data))
-    //     favoriteButton.textContent = "Remove the Favorite"
-    //     favImg.setAttribute('src', "./assets/icons/favorite/favorite-yellow.png")
-    //   }
-    //   else if(favoriteButton.textContent == "Remove the Favorite"){
-    //     myStorage.removeItem(title)
-    //     favoriteButton.textContent = "Favorite the Recipe"
-    //     favImg.setAttribute('src', "./assets/icons/favorite/favorite-blank.png")
-    //   }
+    //set favorite Button functionality
+    //when button is click it is either favorite the recipe 
+    //or it will unfavorite the recipe
+    favoriteButton.addEventListener('click', (event) => {
+      console.log(favImg.alt)
```

Comment:
Should we remove this `console.log()` line?

Reply:
oh yeah.